### DISCUSSION TITLE
Do you plan make Artifacts avaialbe for Agents?

### DISCUSSION THREAD

**Discussion by ogipogi** [danny-avila/LibreChat#4989] [General]
Hi @danny-avila,
Do you plan to make the Artifacts feature available for Agents?
I’ve seen a lot of progress with the Agents system lately, and it would be great to have the Artifacts feature there as well.
https://github.com/danny-avila/LibreChat/discussions/4582 
Thanks for checking

**Comment by danny-avila** [comment]
Yes, I plan to add it as a capability!
